src/radiation/volumeRadiation.cpp

```diff
@@ -4,7 +4,9 @@
 ablate::radiation::VolumeRadiation::VolumeRadiation(const std::string& solverId1, const std::shared_ptr<domain::Region>& region, std::shared_ptr<domain::Region> fieldBoundary,
                                                     const PetscInt raynumber, const std::shared_ptr<parameters::Parameters>& options,
                                                     std::shared_ptr<eos::radiationProperties::RadiationModel> radiationModelIn, std::shared_ptr<ablate::monitors::logs::Log> log)
-    : Radiation(solverId1, region, fieldBoundary, raynumber, options, radiationModelIn, log), CellSolver(solverId1, region, options) {}
+    : Radiation(solverId1, region, fieldBoundary, raynumber, options, radiationModelIn, log),
+      CellSolver(solverId1, region, options),
+      solverRegionMinusGhost(std::make_shared<domain::Region>(solverId1 + "_minusGhost")) {}
```

Comment:
Is the solverRegionMinusGhost being used anywhere?

Reply:
Ah, I had set the VolumeRadiation up to omit the ghost cells. During debugging I realized it made no difference so I removed it. I can get rid of that label.